package.json

```diff
@@ -79,7 +79,7 @@
     "eslint-import-resolver-node": "^0.3.1",
     "eslint-module-utils": "^2.2.0",
     "has": "^1.0.1",
-    "lodash": "^4.17.4",
+    "lodash": "^4.17.11",
```

Comment:
This is unnecessary, since `^4.17.4` is a semver range that includes `4.17.11` already. There's no CVE to resolve.

Reply:
I agree, it includes, but version `4.17.4` has vulnerability, so there should be no possibility to install this version, in order to prevent installing buggy code. Leaving it as `^4.17.4` still allows the vulnerable version to be installed (and it's already happening in one of my library that uses `eslint-plugin-import`). For a test, I forked and installed it locally with this change and it resolves the issue. So, at least we should bump this version to `^4.17.5`

Reply:
If you install a newer version of lodash at the top level, it will dedupe across this package, since it's in the compatible version range. Separately, the CVE doesn't apply to this package's usage of lodash - so there's no actual issue here.

Reply:
Just wanted to clarify this with you @ljharb : I realise that it's been a while since you looked at this.

I have a project using eslint-plugin-import, and when I run npm audit  it identifies this package as introducing a version of lodash vulnerable to prototype pollution - but to fix this issue I need to install a new version of lodash - so I'd create a dependancy in my project to lodash (even though it has no dependancy) 

Services such as Snyk would - in all likelihood - identify this as a potential risk - but the way to mitigate that is to replace what was installed with this library to a newer version through my consuming project

Reply:
Since you have a lockfile in order to use npm audit, you can update the version of lodash without adding it to package.json using npm audit’s suggested commands.